tasks/upload.py

```diff
@@ -712,10 +718,6 @@ def _schedule_test_results_processing_task(
                 )
                 processor_task_group.append(sig)
         if processor_task_group:
-            checkpoint_data = None
-            if checkpoints:
-                checkpoints.log(UploadFlow.INITIAL_PROCESSING_COMPLETE)
```

Comment:
I don't see initial processing complete anymore... has it been moved to another task or are we not using it?

I'm not comfortable approving the PR without knowing why this has been removed. So I will defer approval to someone that knows better

Reply:
this is happening in the test_results scheduling and that's why I removed it, since I don't initialize the `checkpoints` anyways when the report type is `test_results`

Reply:
I see. Thanks for the update here.